files/image_config/kubernetes/scripts/kube_reset.sh

```diff
@@ -0,0 +1,72 @@
+#! /bin/bash
+
+# Redirect any stdout & stderr into logger
+#
+exec 1> >(logger -s -t kube_reset_stdout) 2>&1
+
+do_reset=false
+test_only=false
+
+while getopts ":ft" opt
+do
+    case ${opt} in
+        f ) # Force a reset
+            do_reset=true
+            ;;
+        t ) # Force a reset
+            test_only=true
+            ;;
+        \? ) cat << EOF
+Usage: [-f]
+
+    -f -- Force a reset
+    -t -- test only. Reset action would be skipped
+EOF
+            exit -1
```

Comment:
Suggest to create a `usage()` function and call that here. More readable.

Reply:
ok.